src/main/java/com/revature/RevRelay/services/UserService.java

```diff
@@ -188,6 +190,41 @@ public UserDTO updateUser(String token, UserDTO userDTO) throws UsernameNotFound
         return new UserDTO(userRepository.save(user));
     }
 
+    /**
+     * Borrows the method from NL, overloaded to take updateUserDTO - AL
+     * See above comment, Doesn't allow for null entries
+     * Method overloads for userUpdate to update a user from the frontend
+     *
+     * @param token   JWT corresponding to a User in the database.
+     * @param userDTO UserDTO deserialized from a Controller query.
+     * @return UserDTO object corresponding to the User after edits.
+     * @throws UsernameNotFoundException If Token fails to find a User
+     */
+    public UserDTO updateUser(String token, UserUpdateDTO userDTO) throws UsernameNotFoundException {
```

Comment:
If this method takes a UserUpdateDTO as input, it should give a UserUpdateDTO as output.